javascript/javascript1/week3/noteTakingapp.js

```diff
@@ -0,0 +1,36 @@
+//save a note
+
+function saveNote(contentInput, idInput) {
+const obj = {
+            content: contentInput,
+            id:idInput
+        }
+    notes.push(obj);
```

Comment:
There is a shorter way here.
```
function saveNote(content, id) {
  const obj = {
            content,
            id,
        }
----
```
https://javascript.plainenglish.io/object-literals-using-object-property-shorthand-6360825c60ef